src/tabs/interfaces.ts

```diff
@@ -11,6 +12,14 @@ export interface TabsProps extends BaseComponentProps {
    * - `label` (ReactNode) - Tab label shown in the UI.
    * - `content` (ReactNode) - (Optional) Tab content to render in the container.
    * - `disabled` (boolean) - (Optional) Specifies if this tab is disabled.
+   * - `dismissible` (boolean) - (Optional) Determines whether the tab includes a dismiss icon button. By default, the dismiss button is not included.
+   * - `dismissLabel` (boolean) - (Optional) Specifies an aria-label for the dismiss icon button.
+   *    When a user clicks on this button the onDismiss handler is called.
```

Comment:
I think this belongs to `dismissible` not to `dismissLabel`? (or could even be skipped entirely, I think it's fairly self-explanatory)

Reply:
Ah yea that looks like a typo - I removed it in the next commit since I agree it's self-explanatory + is described elsewhere in our guidelines / implementation